### PR TITLE
Page People, filter is_archived hosts, mobile improvements

### LINKED ISSUE #92: Hosts, Guests - implement filtering on listing page
Could be useful to present a simple filter on large Hosts, Guest listing pages.

could potentially relate to #26

### LINKED ISSUE #262: People page - Add pagination to People page
https://jupiterbroadcasting.com/people/

- paginate peoples page (tl;dr: not straightforward)
  - references:
    - [Paginating PageGroups](https://gohugo.io/templates/pagination/#list-paginator-pages)
  - When using this: `{{ range (.Paginate ((where site.RegularPages "Section" "people").GroupBy "Type").Reverse ) }}`
    - errors received where
      - no paginator
image
      - our custom paginator
image
      - the hugo internal paginator
image

_Originally posted by @elreydetoda in https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues/252#issuecomment-1210219425_

### PR DESCRIPTION
**Pull request by ChanceM**
Resolves #262 
Resolves #92 

- Adds filtering for is_archived
- Pages guests on People Page
- Add mobile rendering improvements

### CODE REVIEW

**Comment by ChanceM** [comment]
JupiterBroadcasting/show-scraper#41 should be merged along with this PR to prevent data being overwritten by the scraper.

File: themes/jb/layouts/partials/people/horizontal.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<div class="card is-horizontal">
-  <div class="columns">
-    <div class="column is-one-third pr-0">
+<div class="card is-horizontal pt-4">
+  <div class="columns section">
+    <div class="column is-one-third pr-0 px-0 pt-0 pb-1 columns mb-0">
```
Comment (CGBassPlayer) [inline themes/jb/layouts/partials/people/horizontal.html:3]: `px-0` should cover what `pr-0` is doing
```suggestion
    <div class="column is-one-third px-0 pt-0 pb-1 columns mb-0">
```

**Review by CGBassPlayer — CHANGES_REQUESTED** [review]
Loving the improvements!

Only question I have is, should we add the archived flags to [Joe](https://www.jupiterbroadcasting.com/hosts/joe)
 and [Drew](https://www.jupiterbroadcasting.com/hosts/drew-devore)? Joe's last show was Dec 2020 and Drews was Mar 2020. I would be fine with leaving Drew active as he still edits for the shows, but just want to throw that out here since we are cleaning up the page.

**Review by CGBassPlayer — APPROVED** [review]
LGTM

**Pull request merged**